src/briefcase/platforms/linux/system.py

```diff
@@ -0,0 +1,721 @@
+                f"{app.packaging_format.upper()} format."
+            )
+
+    def _package_deb(self, app: AppConfig, **kwargs):
+        self.logger.info("Building .deb package...", prefix=app.app_name)
+
+        package_path = self.bundle_path(app) / "package"
+
+        # The long description *must* exist.
+        if app.long_description is None:
+            raise BriefcaseCommandError(
+                "App configuration does not define `long_description`. "
+                "Debian projects require a long description."
+            )
+
+        # Write the Debian metadata control file.
+        with self.input.wait_bar("Write Debian package control file..."):
+            if (package_path / "DEBIAN").exists():
+                self.tools.shutil.rmtree(package_path / "DEBIAN")
+
+            (package_path / "DEBIAN").mkdir()
+
+            # Add runtime package dependencies. App config has been finalized,
+            # so this will be the target-specific definition, if one exists.
+            # libc6 is added because lintian complains without it, even though
+            # its a dependency of the thing we *do* care about - python.
+            system_runtime_requires = ", ".join(
+                [
+                    f"libc6 (>={app.glibc_version})",
+                    f"python{app.python_version_tag}",
+                ]
+                + getattr(app, "system_runtime_requires", [])
+            )
+
+            with (package_path / "DEBIAN" / "control").open("w", encoding="utf-8") as f:
+                f.write(
+                    "\n".join(
+                        [
+                            f"Package: { app.app_name }",
+                            f"Version: { app.version }",
+                            f"Architecture: { self.linux_arch }",
+                            f"Maintainer: { app.author } <{ app.author_email }>",
+                            f"Homepage: { app.url }",
+                            f"Description: { app.description }",
+                            f" { debian_multiline_description(app.long_description) }",
+                            f"Depends: { system_runtime_requires }",
+                            f"Section: { getattr(app, 'system_section', 'utils') }",
+                            "Priority: optional\n",
+                        ]
+                    )
+                )
+
+        with self.input.wait_bar("Building Debian package..."):
+            try:
+                # Build the dpkg.
+                self.tools[app].app_context.run(
+                    ["dpkg-deb", "--build", "--root-owner-group", "package"],
+                    check=True,
+                    cwd=self.bundle_path(app),
+                )
+            except subprocess.CalledProcessError as e:
+                raise BriefcaseCommandError(
+                    f"Error while building .deb package for {app.app_name}."
+                ) from e
+
+            # Move the deb file to it's final location
```

Comment:
typo
```suggestion
            # Move the deb file to its final location
```

Reply:
I *swear* I know the rules around its/it's and auto carrot is having a merry old time with me....